gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/GradeStatisticsPanel.java

```diff
@@ -280,6 +316,11 @@ private double calculateVariance(final List<Double> allGrades) {
 	private double calculateStandardDeviation(final List<Double> allGrades) {
 		return Math.sqrt(calculateVariance(allGrades));
 	}
+
+	private boolean isExtraCredit(Double grade, Assignment assignment) {
+		return (GbGradingType.PERCENTAGE.equals(gradingType) && grade > 100) ||
```

Comment:
Yeah it does seem to work.  The `grade` comes in already scaled as a percentage, so 75% (or 150/200) comes in as 75.0.